src/akamai/lib/ExternalJavaScriptAPI.as

```diff
@@ -85,7 +85,6 @@ package
       addEventListener("setVideoClosedCaptions", onSetVideoClosedCaptions);
       addEventListener("setVideoClosedCaptionsMode", onSetVideoClosedCaptionsMode);
       addEventListener("setTargetBitrate", onSetTargetBitrate);
-      addEventListener("setInitialTime", onSetInitialTime);
```

Comment:
Yes, we are calling setInitialTime in seek event. So no need of separate setInitialTime event